Tests/FluentAssertions.Shared.Specs/XDocumentAssertionSpecs.cs

```diff
@@ -17,7 +17,7 @@ public class XDocumentAssertionSpecs
         #region Be / NotBe
 
         [TestMethod]
-        public void When_asserting_a_xml_document_is_equal_to_the_same_xml_document_it_should_succeed()
+        public void When_asserting_a_XDocument_is_equal_to_the_same_XDocument_it_should_succeed()
```

Comment:
Actually, the principle is to not tie test names to *any* literal name of a code element or anything alike. Next to that, the name should be functional. So in this particular case, I think I would name the test `When_an_xml_document_is_equal_to_another_it_should_not_throw`. But the problem is that I don't really know what *equals* means in this context. Otherwise I would make it more specific.